Sources/swiftpm-testing-helper/Entrypoint.swift

```diff
@@ -0,0 +1,49 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2014-2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Darwin.C
+
+@main
+struct Entrypoint {
+    static func main() throws {
+        let args = CommandLine.arguments
+        if args.count >= 3, args[1] == "--test-bundle-path" {
```

Comment:
We can simplify this to just look at `args[1]` and not have `--test-bundle-path`, if you like?

Reply:
Done!